## Description: Install PeerVPN binary into /usr/bin (as opposed to /usr/local/sbin)
## Origin/Author: Sergei Kolobov <sergei@kolobov.com>
## Bug: bug URL
Index: peervpn-0-044/Makefile
===================================================================
--- peervpn-0-044.orig/Makefile
+++ peervpn-0-044/Makefile
@@ -7,6 +7,7 @@ peervpn: peervpn.o
 peervpn.o: peervpn.c
 
 install:
-	install peervpn /usr/local/sbin/peervpn
+	mkdir -p $(DESTDIR)/usr/bin
+	install -m 755 -o root -g root peervpn $(DESTDIR)/usr/bin/peervpn
 clean:
 	rm -f peervpn peervpn.o
